src/widgets/List.tsx

```diff
@@ -45,6 +45,8 @@ interface ListItemPropsType {
   href?: string;
   insetChildren?: boolean;
   nestedItems?: React.ReactNode[];
+  selected?: boolean;
+  customClass?: any;
```

Comment:
Why not call it `className` and propagate that? (including having the correct type for it)? It's essentially what you are doing just with a non-standard name.